server/src/routers/converterRouter.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+const { parseInputFile, sendResponse } = require('../util');
+
+router.use(express.json());
+
+var Converter = require('api-spec-converter');
+
+//Endpoint for conversion
+router.post('/api/specification/:format/:convertTo', upload.single('file'), (req, res) => {
+    const {format, convertTo} = req.params,
+        {targetSyntax = 'json', toFile = false} = req.query;
+
+    // Check if the conversion is possible
+    if(!(mapping[format] && mapping[format].includes(convertTo))) {
+      return res.status(404).send({'message': 'Conversion mapping not found.'});
+    }
+    else {
```

Comment:
Is this else necessay? 
if conversion is not possible, you anyway will do the rest of the stuff mentioned. Remove the else and try running your code @pcgamer1